packages/auth/ios/RNFBAuth/RNFBAuthModule.m

```diff
@@ -91,7 +93,7 @@ - (void)invalidate {
     [[FIRAuth authWithApp:firebaseApp] removeIDTokenDidChangeListener:[idTokenHandlers valueForKey:key]];
   }
   [idTokenHandlers removeAllObjects];
-  
+  [mfaResolvers removeAllObjects];
   [credentials removeAllObjects];
```

Comment:
nitpick - just for consistency, the rest of the code appears to work on mfaResolvers right after credentials, makes it easier to scan for consistent handling

```suggestion
  [credentials removeAllObjects];
  [mfaResolvers removeAllObjects];
```